### PR TITLE
Fix broken pipeline (Add null checks to custom footers)

### PR DESCRIPTION
**Pull request by mmafe**
# Summary of changes
Pipeline is broken due to null content in custom footer. Adding null checks to footer.

## Frontend
Adding null checks to footer.

[x] My changes are accessible (at minimum WCAG 2.0 Level AA)
[x] My changes are responsive and appear as expected on mobile and desktop views.
[n/a] After merging, I will update the [Content Hub documentation](https://uoguelphca.sharepoint.com/sites/UniversityContentHubInformationGroup) and ensure the Content Hub trainer understands how my changes will affect users.

## Backend
None

# Test Plan

Check that Netlify gus pipeline can complete for this pull request.

### CODE REVIEW

File: src/components/shared/customFooter.js [hunk 1]
```diff
@@ -44,9 +44,9 @@ const DefaultFooter = ({logos, text, widgets}) => (
 
 const CustomFooter = (props) => {
   const footer = props.footerData;
-	const footerLogos = (contentExists(footer.node.relationships.field_footer_logo) ? footer.node.relationships.field_footer_logo : null);
-	const footerText = (contentExists(footer.node.body.processed) ? footer.node.body.processed : null);
-	const footerWidgets = (contentExists(footer.node.relationships.field_widgets) ? footer.node.relationships.field_widgets : null);
+	const footerLogos = (contentExists(footer.node.relationships?.field_footer_logo) ? footer.node.relationships.field_footer_logo : null);
+	const footerText = (contentExists(footer.node.body?.processed) ? footer.node.body.processed : null);
+	const footerWidgets = (contentExists(footer.node.relationships?.field_widgets) ? footer.node.relationships.field_widgets : null);
```
Comment (tqureshi-uog) [inline src/components/shared/customFooter.js:50]:
```suggestion
    const footerLogos = footer.node.relationships?.field_footer_logo ? footer.node.relationships.field_footer_logo : null;
    const footerText = footer.node.body?.processed ? footer.node.body.processed : null;
    const footerWidgets = footer.node.relationships?.field_widgets ? footer.node.relationships.field_widgets : null;
```

File: src/components/shared/customFooter.js [hunk 2]
```diff
@@ -28,9 +28,9 @@ const DefaultFooter = ({logos, text, widgets}) => (
       <div className="col-md-3 content-area">
```
Comment (tqureshi-uog) [inline src/components/shared/customFooter.js:28]: Remove unneeded call to `contentExists` in line 27

**Review by tqureshi-uog — CHANGES_REQUESTED** [review]
I've been meaning to remove the `contentExists` function from our code because there are far more efficient ways to check for null. This would be a good time to purge it from this component since the null checks have been updated to the proper format.

File: src/components/shared/customFooter.js (hunk 2 above)
Comment (mmafe) [reply src/components/shared/customFooter.js:27]: Removed contentExists

File: src/components/shared/customFooter.js (hunk 1 above)
Comment (mmafe) [reply src/components/shared/customFooter.js:49]: Removed unnecessary parentheses.

**Comment by mmafe** [comment]
Spoke too soon...one looks like it finished building. Hopefully it deploys.

**Review by tqureshi-uog — APPROVED** [review]
https://deploy-preview-175--ugconthub.netlify.app/ completed successfully

**Pull request merged**